qna/DE-certified-online-TSE.md

```diff
@@ -1,8 +1,24 @@
 ## Question
-Is there any certified online TSE available for germany?
+Is there any certified Cloud TSE available for germany?
+
+## Question
+
+I want to roll out with a Cloud TSE and  i do not want to wait any longer. What options do i have?
+
+## Question
+
+I have ordered a Cloud TSE from my dealer. Can i use an uncertified Cloud TSE in my region until a certified version is available?
 
 ## Metadata tags
 lang-en, market-de, middleware, PosCreator, PosDealer, Consultant
 
 ## Answer
-Currently (18.08.2020), there is **no certified** online TSE available for the german market. 
+Currently (23.09.2020), there is **no certified** Cloud TSE available for the german market. fiskaltrust will  support certified Cloud TSE's for production use as soon as they are available on the market. In the meanwhile, we offer uncertified Cloud TSEs for integration and testing in the sandbox environment.
+
+PosDealers can already order fiskaly Cloud TSEs from fiskaltrust by [signing a frame contract.](https://github.com/fiskaltrust/productdescription-de-doc/blob/8a9c3fa62c9dd0c38662c7127a724162703242e3/for-posdealers/02-pre-sales/01-purchase-agreement.md)
+
+If a PosOperator wants to be compliant in production, we recommend to use certified Hardware TSE's.
+
+In general, if PosOperators are using an uncertified TSE in production or no TSE at all, it will both result in generating pos receipts which are not compliant. In some regions of germany it may be possible to [extend the "Nichtbeanstandungsregelung" under some specific circumstances](https://github.com/fiskaltrust/faq/blob/master/qna/DE-Nichtbeanstandungsregelung.md#regionale-erg%C3%A4nzungen-zur-nichtbeanstandungsregelung). We recommend to refer to a tax advisor to ensure to fulfil all requirements to stay compliant.
+
+If all conditions for regional extensions of the "Nichtbeanstandungsregelung" are met, it is possible to already use the fiskaltrust.Middleware to prepare a Cloud-TSE rollout. In this case, it may be possible to start without Cloud TSE, SCU and initial receipt, but already profiting from the Security Mechanism of fiskaltrust and collecting data for DSFinV-K export.
```

Comment:
```suggestion
If all conditions for regional extensions of the "Nichtbeanstandungsregelung" are met, it is possible to already use the fiskaltrust.Middleware to prepare a Cloud-TSE rollout. In this case, it may be possible to start without Cloud TSE, SCU and initial receipt, but already profiting from the  of fiskaltrust.SecurityMechanism and collecting data for DSFinV-K export.
```